test/performance/README.md

```diff
@@ -1,128 +1,123 @@
-## Stress Test Experiments
+## Workload Benchmark for KubeStellar
 
+*Pre-requisite*: in order to follow the instructions below, you must have an environment with KubeStellar installed; see [KubeStellar getting started](https://docs.kubestellar.io/release-0.23.1/direct/get-started/). Alternatively, you can also use KubeStellar e2e script [run-test.sh](https://github.com/kubestellar/kubestellar/blob/main/test/e2e/run-test.sh) to setup an environment.
 
-## Generate the workload traffic
+To generate the sample workload for KubeStellar performance experiments proceed as following:
 
-1. Clone the following fork of the clusterloader2 repo: 
+### Generate the workload traffic
 
-```bash
-$ git clone -b ks-provider https://github.com/dumb0002/perf-tests.git
-```
+1. Clone the following clusterloader2 repo: 
 
-2. Configure the parameters of your experiment:  
+   ```bash
+   $ git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
 
-   a) cd into the load configuration directory
+   Set the variable `CL2_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
 
-```bash
-$ cd  perf-tests/clusterloader2/testing/load
-```
-   b) create the `stress-test` namespace - you workload needs to be deployed into this exact namespace (temporary solution)
-
-```bash
-$ kubectl apply -f - <<EOF
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: stress-test
-  labels:
-    app.kubernetes.io/cluster: wec1
-EOF
-```
-
-   c) configure your sample workload: 
-   
-```bash
-$ vi test-workload.yaml
-```
-  
-   d) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
-   
-```bash
-$ vi performance-test-config.yaml
-``` 
+   ```bash
+   $ export CL2_DIR=$HOME/perf-test/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) cd into the `test` directory from your local copy of the KubeStellar repo, for example:
```

Comment:
The command shown below does not `cd` into the `test` directory, it goes into `test/performance`.

Reply:
Oh, I see! that was a typo, just fixed it!